chatbot/admin.py

```
@@ -0,0 +1,303 @@
+"""
+پنل مدیریت سیستم چت‌بات
+Chatbot Admin Panel
+"""
+
+from django.contrib import admin
+from django.utils.html import format_html
+from django.urls import reverse
+from django.utils.safestring import mark_safe
```

Ruff (0.12.2)

`django.utils.safestring.mark_safe` imported but unused

Remove unused import: `django.utils.safestring.mark_safe`

(F401)